src/critics/path_angle_critic.cpp

```diff
@@ -1,15 +1,26 @@
 // Copyright 2022 @artofnothingness Alexey Budyakov, Samsung Research
```

Comment:
The naming here makes it quite difficult to understand intent. I propose

- activate_if_angle_to_furthest_more_than_threshold -> ??
- activate_if_path_reached_ratio_less_than_threshold -> max_path_ratio
- pose_point_angle -> poseToGoalAngle (the name otherwise is not clear; rename x and y to `goal_x` and `goal_y`)
- pathRatioReached -> `getpathRatio` (since its not making the "reached" determination in that function)

Reply:
> pose_point_angle -> poseToGoalAngle

It's general utility, why use "goal" here ? It's pose (have angle) to point (no angle) angle

> pathRatioReached -> getpathRatio

It's not making it but it returns path ratio that reached. getPathRatioReached will be more precise

Reply:
> It's general utility, why use "goal" here ?

Because that's the only way its being used -- where the point is the goal and makes the code more self documenting.

`getPathRatioReached` :+1: